src/pynwb/form/build/map.py

```diff
@@ -662,7 +680,7 @@ def __copy__(self):
         ret.merge(self)
         return ret
 
-    def __deepcopy__(self):
+    def __deepcopy__(self, memo):
```

Comment:
All of a sudden my tests started needing this argument in deepcopy.  Doesn't hurt anything, though